[2021-10-16 13:27:03,262] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:24:00+00:00 [queued]>
[2021-10-16 13:27:03,286] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:24:00+00:00 [queued]>
[2021-10-16 13:27:03,286] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:27:03,286] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:27:03,287] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:27:03,380] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_1> on 2021-10-16 13:24:00+00:00
[2021-10-16 13:27:03,394] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_1', 'scheduled__2021-10-16T13:24:00+00:00', '--job-id', '634', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmp05nmf3yr', '--error-file', '/tmp/tmpktmeugte']
[2021-10-16 13:27:03,395] {standard_task_runner.py:77} INFO - Job 634: Subtask get_gross_year_1
[2021-10-16 13:27:03,388] {standard_task_runner.py:52} INFO - Started process 2148 to run task
[2021-10-16 13:27:03,466] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_1 scheduled__2021-10-16T13:24:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:27:03,557] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_1
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T13:24:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T13:24:00+00:00
[2021-10-16 13:27:03,570] {python.py:152} INFO - Done. Returned value was: ['$42.05 ', '2010\n']
[2021-10-16 13:27:03,608] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_1, execution_date=20211016T132400, start_date=20211016T132703, end_date=20211016T132703
[2021-10-16 13:27:03,664] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:27:03,776] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
